doc/ug/dv_methodology/index.md

```diff
@@ -299,60 +299,115 @@ Once the regression tool is developed, it will provide a way to enable these cap
+These constructs also allow us to capture whether multiple scenarios have occurred simultaneously through crosses.
+This is also often referred to as cross coverage.
+These constructs are typically encapsulated in a class and are sometimes referred to as the ‘functional coverage model’.
+They are sampled in ‘reactive’ components of the testbench, such as monitors and/or the scoreboards.
 They can also be embedded within the RTL to sample interesting scenarios through DV stimulus.
 
 Here are the metrics used with a brief explanation:
 
-*  **Assert Coverage**: This metric measures which assertions, cover properties and sequences have been exercised in the course of the simulation.
+* **Assertion Coverage using SVA**: This metric measures which assertions have been exercised in the course of the simulation.
+SVA coverage often implemented by Design engineer and is used to add cover procedures and variables not visible on output pins.
   Note, an assertion precondition counts as a cover point.
-*  **Covergroup Coverage**: This metric (sometimes called "Testbench Coverage") measures covergroups during simulation.
-  These are usually coded in the testbench to check that the desired stimulus was generated properly, but can also be embedded in the RTL.
+* **Covergroup Coverage**:Observes values on buses, registers and so on.
+  This can verify that a specific value or range of values was seen on a bus, And that no illegal values were seen.
+  Simulators will throw an error if an illegal bin is filled.
+  Could also be the intersection of two cover points.
+  One example could be observing the FIFO full signal and the FIFO write enable signal is asserted at the same time.
+  This is called a coverage cross and will give valuable information on whether this scenario was ever seen. 
+* **Cover Property coverage**: This observes events or series of events.
+  As an example think of the TL UL register bus used.
+  A cover property cover point could be the handshaking between valid and ready.
+  There is a big overlap in what can be covered by SVA and Cover properties.
+  Suggestions on when to use which form of coverage is presented later in this document.
+
+
+#### Do we need both types of coverage
+
+Reaching a 100% code coverage does not tell you the whole story.
+Think of the simple example of an AND gate. 
+Input A, B, and output X. to get to a 100 percent code coverage only two different input combinations are needed.
+00 and 11, these two will produce all possible outcomes of X.
+
+![single AND gate](dv_method_single_gate.svg)
+
+
+The coverage will indicate that all code was exercised.
+But We do not know that our design work as intended. 
+All we know is that A, B, and X have been observed to take on both logic  0 and 1.
+We could not say for certain that the design was in fact an AND gate could just as easily be an OR gate.
```

Comment:
do you want the colon in there?

Reply:
Yes, because you need something to separate the two clauses. Equally correct (but a bit stuffy looking) would be a semicolon. Or you could split the sentence in two.

Reply:
ok fixed, not sure I like splitting them so added the colon